### PR TITLE
 AST-2243 - Moved border width from pro to free

### PR DESCRIPTION
**Pull request by nathanrodrigues2111**
### Description
Jira : https://brainstormforce.atlassian.net/browse/AST-2243

### Screenshots
https://d.pr/HWp4pg

### Types of changes
Refactor 

### How has this been tested?
- Checked when astra addon is enabled / disabled 
- Checked after resetting theme 
- Checked changing svg icons

### Checklist:
- [x] My code is tested
- [x] My code passes the PHPCS tests
- [x] My code follows accessibility standards 
- [x] My code has proper inline documentation 
- [ ] I've included any necessary tests 
- [ ] I've included developer documentation 
- [ ] I've added proper labels to this pull request

### CODE REVIEW

File: inc/builder/type/header/woo-cart/dynamic-css/dynamic.css.php [hunk 1]
```diff
@@ -429,11 +429,13 @@ function astra_hb_woo_cart_dynamic_css( $dynamic_css, $dynamic_css_filtered = ''
 		
 		if ( ! astra_has_pro_woocommerce_addon() && 'outline' === $header_cart_icon_style && 'default' !== $header_woo_cart_list ) {
```
Comment (imnavanath) [inline inc/builder/type/header/woo-cart/dynamic-css/dynamic.css.php:430]: Do we need this condition - `! astra_has_pro_woocommerce_addon()` ?

Comment (nathanrodrigues2111) [reply inc/builder/type/header/woo-cart/dynamic-css/dynamic.css.php:430]: Yes , As this should only run when astra addons is not enabled @imnavanath

File: inc/core/builder/class-astra-builder-options.php [hunk 2]
```diff
@@ -950,11 +950,13 @@ function astra_hf_builder_customizer_defaults( $defaults ) {
 		'tablet'  => '',
 		'mobile'  => '',
 	);
+	$defaults['woo-header-cart-icon']                      = 'default';
```
Comment (imnavanath) [inline inc/core/builder/class-astra-builder-options.php:953]: If not required we can remove this

Comment (nathanrodrigues2111) [reply inc/core/builder/class-astra-builder-options.php:953]: This is required. I faced bug here. When astra theme is resettled the value of woo-header-cart-icon becomes null / "" and not default and so it was not updating the  woo-header-cart-icon value properly. The outline and border radius were not working as woo-header-cart-icon was empty as when the site is resettled

**Review by imnavanath — APPROVED** [review]

**Pull request merged**